canonical-webteam/global-nav
# Design review changes

## Done
Fixed the design review comments.

## QA
- Pull down branch and run `gulp build`
- Open index.html
- Check that all the points in [the issue](https://github.com/canonical-webteam/global-nav/issues/26) have been addressed

## Details
Fixes https://github.com/canonical-webteam/global-nav/issues/26

## Screenshot
More link on a large screen:
![canonical global navigation demo](https://cloud.githubusercontent.com/assets/1413534/25892994/2203dd20-356e-11e7-8a04-ba87c0d87408.png)
